onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -2645,6 +2645,18 @@ def aten_upsample_nearest3d(
     return _aten_upsample_output_size(self, size, "nearest", "asymmetric")
 
 
+@torch_op("aten::upsample_nearest3d.vec", trace_only=True)
+def aten_upsample_nearest3d_vec(
```

Comment:
Is this meant to be same as function above, differing only in how parameter scales is specified?